ocrd/ocrd/cli/workspace.py

```diff
@@ -34,15 +45,15 @@ def __init__(self, directory, mets_basename, automatic_backup):
 # ----------------------------------------------------------------------
 
 @click.group("workspace")
-@click.option('-d', '--directory', envvar='WORKSPACE_DIR', default='.', type=click.Path(file_okay=False), metavar='WORKSPACE_DIR', help='Changes the workspace folder location.', show_default=True)
-@click.option('-M', '--mets-basename', default="mets.xml", help='The basename of the METS file.', show_default=True)
+@click.option('-d', '--directory', envvar='WORKSPACE_DIR', type=click.Path(file_okay=False), metavar='WORKSPACE_DIR', help='Changes the workspace folder location.', show_default=True)
+@click.option('-m', '--mets', default=None, help='The path/URL of the METS file.')
```

Comment:
Now we don't have `show_default` anymore (because the defaults are non-trivially calulcated in WorkspaceCtx), we at least need to write them descriptively into `help`, e.g. `"[default: METS_URL directory or .]"` and `"[default: WORKSPACE_DIR/mets.xml]"`

Reply:
b78e8b8

Reply:
84dc4e1